surveys/surveys.go

```diff
@@ -281,8 +281,7 @@ func GetLauncherSchema(schemaName string, schemaUrl string, cirInstrumentId stri
 	if schemaUrl != "" {
 		log.Println("Getting schema by URL: " + schemaUrl)
 		launcherSchema = LauncherSchema{
-			URL:  schemaUrl,
-			Name: schemaName,
+			URL: schemaUrl,
```

Comment:
I'm not sure why intermittently had success with v1 launcher, but it looks like this is the issue for the v1 launch - because `schemaName` isn't populated anymore, the block of code that you put back isn't going to set the name, which is why v1 launch is still missing it

Reply:
Yeah I thought this would be the issue if there was one. I was waiting to have a chat with you guys before removing this 👍